web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue

```diff
@@ -0,0 +1,141 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="sort-header-container">
```

Comment:
shouldnt this be tr tag and th`s inside?

Reply:
Using Divs is following the same design pattern as the other SortingHeader components.